samples/excel/excel-library/overview/wwwroot/BlazorDownloadFile.js

```diff
@@ -0,0 +1,56 @@
+﻿// these methods are from:
+// https://www.meziantou.net/generating-and-downloading-a-file-in-a-blazor-webassembly-application.htm
```

Comment:
is this source code under copyright licensed?

Reply:
no, but it should be mentioned in the docs